### PR TITLE
docs: fix grammatical errors & links in README of Hindi language

### PR DESCRIPTION
**Pull request by Rahulbaran**
- Fixed markdown errors of links in lines 110, & 156
- Removed some grammatical errors in lines 24, 33, 35, 44, 53 & 56

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.

Closes #XXXXX

### CODE REVIEW

**Review by erictleung — CHANGES_REQUESTED** [review]

File: README-HI.md [hunk 1]
```diff
@@ -41,7 +41,7 @@
 - [मोज़िला का योगदानकर्ता पारिस्थितिकी तंत्र](#mozillas-contributor-ecosystem)
 - [नए ओपन सोर्स योगदानकर्ताओं के लिए उपयोगी लेख](#useful-articles-for-new-open-source-contributors)
 - [संस्करण नियंत्रण का उपयोग करना](#using-version-control)
-- [स्रोत पुस्तकें खोलें](#open-source-books)
+- [ओपन सोर्स पुस्तकें](#open-source-books)
```
[the remark this answers is not in the document]
Comment (Rahulbaran) [reply README-HI.md:44]: I will fix that and make a PR

**Comment by Rahulbaran** [comment]
Changes made in lines 125,127,129 & 135

**Comment by Rahulbaran** [comment]
Someone please check my PR

**Review by erictleung — COMMENTED** [review]
LGTM! 👍 Thanks for your patience. We appreciate your grammatical review and suggestions for our translations ✨

**Review by erictleung — APPROVED** [review]

**Pull request merged**